app/js/model/list_model.js

```diff
@@ -6,6 +6,7 @@ export default class ListModel extends Model {
       'http://fxos.github.io/sharing/dist/app/manifest.webapp': {
         manifestURL: 'http://fxos.github.io/sharing/dist/app/manifest.webapp',
         name: 'sharing',
+        icon: 'https://raw.githubusercontent.com/fxos/sharing/master/app/icons/Icon_P2P_Sharing.png',
```

Comment:
I considered that, but it's trickier for hosted packages in which the icon is specified with a relative link and zipped inside the package. We'd have to download the whole zip to get it. Seem like a lot of bandwidth and big delay for just the icon.